tools/project-context.js

```diff
@@ -596,10 +597,21 @@ _.extend(ProjectContext.prototype, {
     var self = this;
     var searchDirs = [files.pathJoin(self._projectDirForLocalPackages, 'packages')];
 
-    if (! self._ignorePackageDirsEnvVar && process.env.PACKAGE_DIRS) {
-      // User can provide additional package directories to search in
-      // PACKAGE_DIRS (colon-separated).
-      _.each(process.env.PACKAGE_DIRS.split(':'), function (p) {
+    // User can provide additional package directories to search in
+    // METEOR_PACKAGE_DIRS (colon-separated),
+    // This was added after 1.4.0.1
+    var envPackageDirs = process.env.METEOR_PACKAGE_DIRS || process.env.PACKAGE_DIRS;
+
+    // If the PACKAGE_DIRS, warn users to migrate to the new env vars.
+    if (process.env.PACKAGE_DIRS) {
+      console.warn("For compatibility reasons, the 'PACKAGE_DIRS' environment"
+                   + " variable is deprecated and will be removed in a future"
+                   + " release of Meteor.  Developers should now use"
+                   + " 'METEOR_PACKAGE_DIRS'.");
```

Comment:
`console` -> `Console`

Use `+` on the end of the line, and have the space on the end of the line, like:

```
Console.warn('Foo ' +
  'bar.');
```

or simply use `,` to separate, and have no space on the edge of each line:

```
Console.warn('Foo',
  'bar.');
```